src/app/app-maps/components/remove-map/remove-map.component.html

```diff
@@ -12,21 +12,28 @@
 <!--See the License for the specific language governing permissions and-->
 <!--limitations under the License.-->
 
-<mat-dialog-content class="mat-primary" fxLayout="colum" fxFlexAlign="center center">
+<mat-dialog-content
+  class="mat-primary"
+  fxLayout="colum"
+  fxFlexAlign="center center"
+>
   <div class="map-progress">
     <div class="box">
-      <span class="error" [hidden]="!(error | async)">An error ocurred removing the map. Please try again later.</span>
-      <div [hidden]="(error | async)">
-        <div>
-          <mat-icon class="error-icon">error_outline</mat-icon>
-        </div>
-        <span class="text">Are you sure you want to delete {{map.name}}?</span>
+      <span class="error" [hidden]="!(error | async)"
+      >An error ocurred removing the map. Please try again later.</span
+      >
+      <div [hidden]="error | async">
+        <div><mat-icon color="warn" class="error-icon">error_outline</mat-icon></div>
+        <span class="text"
+        >Are you sure you want to delete <i>{{ map.name }}</i>?</span
```

Comment:
One should normally prefer `em` tags to `i`, as the former is more about semantics while the latter is only about presentation, which belongs in css.